### DISCUSSION TITLE
A notification from TGC

### DISCUSSION THREAD

**Discussion by Tetr1sForev3r** [thatskymod/Sky-CotL-Scripts#84] [General]
So in the notification box (in sky) i got this warning saying that the game detected a third party tool? In other tools that I may be hacking the game. Does this endanger my account in any way? Do I need to uninstall the canva?

**Comment by Imrantamrin** [comment]
Is it dangerous for my sky account

Pada Sel, 14 Mei 2024 2.09 PM, Tetris ***@***.***> menulis:

**Comment by Haki-Skykid** [comment]
Do you use any 3rd party software besides Canvas? If you have uses other than Canvas, please uninstall one of them👌
 `__`

**Comment by Hentilador** [comment]
happening to me too, which mod were you all using?

**Comment by thatskymod** [comment]
> Is it dangerous for my sky account

Fun fact regarding account ownership:

**You do not own your TGC Account**, which at all times remains the property of thatgamecompany. TGC may cancel your account at any time, for any reason, or without reason at all.[^1]

[^1]: [EULA / Terms of Service — Sky: Children of the Light Help Center](https://thatgamecompany.helpshift.com/hc/en/17-sky-children-of-the-light/faq/460-eula-terms-of-service/)

**Comment by thatskymod** [comment]
Sorry to hear, @Tetr1sForev3r! It seems you're not the only one affected. Rather, it's likely that TGC used an automated script to scan their systems for suspicious activity.

Did you unlock seasonal quests early? Perhaps you purchased an unreleased item? These could be strong indications of cheating.

Cheating has never been safe to begin with.

Now might be an opportune time to stop using Canvas or any other modifications you employ.

**Reply by Tetr1sForev3r** [reply]
I do not think I ever did get anything earlier than I'm supposed to (except wing buffs, I don't know if that counts though). Recently I've only used the cheats for auto-cr as it's too time consuming, but either way I'll consider uninstalling the canva for now until it's all clear. Thank you for the answer! ♡

**Reply by Hentilador** [reply]
I think what I did to get the message was getting the elder hair before I could even talk to the elders.
